TaskHouseApi/Model/Estimate.cs

```diff
@@ -0,0 +1,44 @@
+using System.ComponentModel.DataAnnotations;
+using Newtonsoft.Json;
+
+namespace TaskHouseApi.Model
+
+{
+    public class Estimate : BaseModel
+    {
+        [Required]
+        public int TotalHours { get; set; }
+        [Required]
+        public int Complexity { get; set; }
+        [Required]
+        public decimal HourlyWage { get; set; }
+        public bool Accepted { get; set; } = false;
+        [Required]
+        public decimal Urgency { get; set; }
+        [Required]
+        public int WorkerId { get; set; }
+        [Required]
+        public int TaskId { get; set; }
+        public decimal PriceEstimate { get; set; }
+
+        public Estimate()
+        {
+
+        }
+
+        public Estimate(int TotalHours, int Complexity, decimal HourlyWage, decimal Urgency)
+        {
+            this.TotalHours = TotalHours;
+            this.Complexity = Complexity;
+            this.HourlyWage = HourlyWage;
+            this.Urgency = Urgency;
+
+            this.PriceEstimate = Calculate();
+        }
+
+        public decimal Calculate()
+        {
+            return TotalHours * HourlyWage * Complexity * Urgency;
+        }
```

Comment:
PriceEstimate can get out of sync if calculate isn’t called before its used. I would recommend to remove PriceEstimate and use Calculate instead when the price is needed.